site/app/templates/autograding/PeerResults.twig

```diff
@@ -9,6 +9,38 @@ $( document ).ready(function() {
         $("#ShowRegradeRequestButton").hide();
     }
 });
+function open_peer_feedback_tab(user, ordered_graders) {
+    $('.page-content').hide();
+    $('#peer-feedback-'+user).show();
+    for (var i = 0 ; i<{{ordered_graders|json_encode|raw}}.length;i++){
```

Comment:
Can you explain exactly what is going on here with your filters?

Reply:
Added Comment explaining